app/components/common/Modal.js

```diff
@@ -66,7 +67,7 @@ Modal.propTypes = {
   body: PropTypes.string,
   footer: PropTypes.string,
   close: PropTypes.func,
-  modal: PropTypes.object.isRequired
+  modal: PropTypes.object
```

Comment:
Just double checking: is this property not required anymore? It seems like the whole purpose of this component is to display a modal window.

Reply:
Replied on the other PR. Thanks